### PR TITLE
Fix agfs fees

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

**Comment by ibrechin** [comment]
These don't fix all the failures, but it's what I've got so far.

File: fee_calculator/apps/calculator/fixtures/price.json [hunk 1]
```diff
@@ -34062,7 +34062,7 @@
     "fee_type": 15,
     "unit": "DAY",
     "fixed_fee": "0.00000",
-    "fee_per_unit": "151.00000",
```
Comment (jsugarman) [inline fee_calculator/apps/calculator/fixtures/price.json]: 👍 certainly looks like its in line with [The Criminal Legal Aid (Remuneration) Regulations 2013](http://www.legislation.gov.uk/uksi/2013/435/schedule/1/paragraph/21/made)

File: fee_calculator/apps/calculator/fixtures/price.json [hunk 2]
```diff
@@ -34808,7 +34808,7 @@
     "advocate_type": "QC",
     "offence_class": null,
     "fee_type": 12,
-    "unit": "HALFDAY",
```
Comment (jsugarman) [inline fee_calculator/apps/calculator/fixtures/price.json]: tallies with these [regulations](http://www.legislation.gov.uk/uksi/2013/435/schedule/1/paragraph/24/made) for "committal for sentence"

screen shot 2018-07-06 at 12 17 23

File: fee_calculator/apps/calculator/fixtures/price.json [hunk 3]
```diff
@@ -35468,7 +35468,7 @@
     "advocate_type": "JRALONE",
     "offence_class": null,
     "fee_type": 12,
-    "unit": "HALFDAY",
```
Comment (jsugarman) [inline fee_calculator/apps/calculator/fixtures/price.json]: just to note the regs do not specify fees for "Junior alone" so i guess we defer to CCR which treats it the same as Led Junior

**Pull request merged**